### ISSUE TITLE
iOS version

### ISSUE THREAD

**Issue by perkstvr** [leggett/simplify#109]
Is there any chance you can make a version compatible with IOS to bring simplify to the iPad where you use the larger screen and Simplify becomes even more beneficial so you don’t have to have unnecessary information on the screen.

I know the focus is on the chrome extension but a standalone app would be great.

**Comment by leggett** [comment]
Not possible. I can't modify mobile apps and mobile browsers (the main ones) don't support extensions.

**Comment by avernet** [comment]
@leggett Time to re-open this one, now that Safari in iOS/iPadOS 15 supports extensions? (And one would hope that Apple makes it really easy to create extensions that work for both Safari on macOS and iOS/iPadOS… but, you never know!) I appreciate your extension on Safari for macOS, and would love to also be able to use it when I'm on my iPad.

**Comment by leggett** [comment]
Maybe? But isn't the Gmail iOS app better than using Gmail in Safari, even with Simplify?

**Comment by avernet** [comment]
Well, I imagine this is matter of preference. On an iPhone, I'd say that the app is much better. But on an iPad, when used with a keyboard, I much prefer the web app: it provides access to a URL pointing to each thread, has more keyboard bindings, more features.

Or, you can reframe the question as follows: if you could run the Gmail iPadOS app on your computer, would you use it instead of the web app? (And in fact, if you have an M1 Mac, I imagine you can.) I suspect some people would, but I certainly would not.

**Comment by leggett** [comment]
You're right. My calculation is that most people still use an iPad as a big iPhone (touch device) in which case I think the mobile app is likely best. But as soon as you bring a keyboard into the mix and/or want to go beyond casual usage, I'd switch to the web app.

**Comment by ezdub** [comment]
Aloha- I request that an iPad extension be reconsidered now.

Gmail looks and works great as a web app on iPad already, I just know the Simplify polish would make the experience golden. 

gMail iOS is ok on the phone but the iPad app UX is far from the comforts of processing in Simplify which my brain anticipates since the iPad has the larger screen and Apple keyboard and we've seen years of apple evolving these OSes towards each other. 

Hopefully the Safari web extension conversion rebuilds easily for modern iPad Pros. Maybe the Apple Neo is an easy target in this too.

I'm willing to contribute a lot of coffee $ and myself as a QA resource.
Thanks for considering again. -Z
